ros_buildfarm/pulp.py

```diff
@@ -305,6 +337,28 @@ def mirror_remote_to_distribution(self, remote_name, distribution_name, dry_run=
         if sync_task.created_resources:
             self._publish_and_distribute(distribution, sync_task.created_resources[0])
 
+    def remove_unused_content(self):
+        delete_task_href = self._core_orphans_api.delete().task
+        delete_task = self._wait_for_task(delete_task_href)
+        print('%s' % (delete_task,))
+
+    def remove_unused_repo_vers(self, distribution_name, dry_run=False):
+        unused_repo_vers = self.enumerate_unused_repo_vers(distribution_name)
+
+        # Start removing the oldest ones first
+        unused_repo_vers.reverse()
+
+        for repo_ver in unused_repo_vers:
+            if repo_ver.endswith('/0/'):
```

Comment:
I'm glad you pointed this out. This was awkward and was kind of a hack. Since the root version can never be deleted, I've updated the enumeration code to omit it and consider it "used". It's a fair amount cleaner now.